### ISSUE TITLE
How to use a single GPU to train deeplabv3 to get best score?

### ISSUE THREAD

**Issue by ALEX13679173326** [open-mmlab/mmsegmentation#306]
I just have one GPU,but the mmseg has four!
I can train my model,but can not get the same mIou as the official mIou.
The other config files are not changed!
So someone can help me?

**Comment by NickChang97** [comment]
What's your batch size. If your batch size is small, you may try to use a small learning rate and use more iterations

**Comment by lxtGH** [comment]
You **can not** reproduce the best score using only one gpu since the limited basesize and crop size.
